src/pois.rs

```diff
@@ -176,6 +282,33 @@ impl IndexedPoi {
             &country_codes,
             langs,
         );
+        let mut self_country_langs_to_add: Vec<String> = Vec::new();
+        for country_code in country_codes.iter() {
+            if let Some(langs) = COUNTRIES_LANGS.get(country_code) {
+                for lang in langs {
+                    if !self.poi.labels.0.iter().any(|x| x.key == *lang) {
+                        self_country_langs_to_add.push((*lang).to_owned());
+                    }
+                }
+            }
+        }
+        for lang in self_country_langs_to_add
+            .into_iter()
+            .filter(|l| langs.iter().any(|lang| lang == l))
+        {
+            self.poi.labels.0.push(Property {
+                key: lang,
+                value: format!(
+                    "{} ({})",
+                    self.poi.name,
+                    self.poi
+                        .administrative_regions
+                        .first()
+                        .map(|admin| admin.label.to_string())
+                        .unwrap_or_else(String::new)
+                ),
```

Comment:
I think it would be easier and more consistent to reuse `self.poi.label` here.

Actually using the first administrative_region is probably incorrect, as we have no guarantee that the city is the first element among all admins.